examples/helloapache/Nulecule.yaml

```diff
@@ -0,0 +1,80 @@
+# Using helloapache as an example
+---
+metadata:
+  name: helloworld 
+  id: helloworld
+  appversion: 0.0.1
+  spec: 1.0.0
+  description: Hello world!
+
+provider:
```

Comment:
If we do `provider` we could eliminate having to have `provider-token` to `token`, etc.